keyboards/jisplit89/keymaps/default/config.h

```diff
@@ -0,0 +1,22 @@
+/* Copyright 2020 Salicylic_acid3
```

Comment:
I think the defaults should be used here, which means we might as well delete this file.